subscription-sender/internal/queue/consumer.go

```diff
@@ -0,0 +1,25 @@
+package queue
+
+import (
+	"github.com/nats-io/nats.go"
+
+	"github.com/RinaDish/subscription-sender/internal/services"
+	"github.com/RinaDish/subscription-sender/tools"
+)
+
+type Queue struct {
+	QueueConn *nats.Conn
+	subscriptionService services.SubscriptionService
```

Comment:
Why does Queue depend on concrete value of subscription service?

Reply:
done